ChatCompletionCustomSkill/README.md

```diff
@@ -4,7 +4,7 @@
 
 Quickstart Guide:
 
-- This folder illustrates how to leverage Azure AI Studio to summarize a large piece of text using [custom web api skills](https://learn.microsoft.com/en-us/azure/search/cognitive-search-custom-skill-web-api).
+- This folder illustrates how to leverage Azure AI Studio with a deployed language model of your choice to do a chat-completion skill using [custom web api skills](https://learn.microsoft.com/en-us/azure/search/cognitive-search-custom-skill-web-api).
```

Comment:
change to AI Foundry?

Reply:
whoops, I forgot about that name change!